DataProcessing/datax-host/src/main/scala/datax/input/BatchBlobInputSetting.scala

```diff
@@ -33,8 +33,8 @@ object BatchBlobInputSetting {
 
     InputBlobsConf(
       path = KeyVaultClient.resolveSecretIfAny(dict.getOrNull(SettingPath)),
-      startTime = dict.getOrNull(SettingProcessStartTime),
-      endTime = dict.getOrNull(SettingProcessEndTime),
+      startTime = dict.getOrElse(SettingProcessStartTime, sys.env.getOrElse("process_start_datetime", "")),
```

Comment:
load process_start_datetime and process_end_datetime from system variables if they are not configured in conf file